docs/adrs/rework-batch-analysis-architecture.md

```diff
@@ -0,0 +1,82 @@
+# Rework Batch Analysis Architecture
+
+Date: 2020-09-28
+
+The current implemetnation of Batch Analysis at KBase has the following issues:
```

Comment:
typo - implementation

Reply:
thanks for the catch! there's my dyslexia cropping up